test/new-e2e/tests/agent-metric-logs/log-agent/utils/file_tailing_utils.go

```diff
@@ -115,35 +132,35 @@ func checkLogs(s *LinuxFakeintakeSuite, service, content string, expectLogs bool
 	}, 2*time.Minute, 10*time.Second)
 }
 
-// cleanUp cleans up any existing log files (only useful when running dev mode/local runs).
-func (s *LinuxFakeintakeSuite) cleanUp() {
-	t := s.T()
+// CleanUp cleans up any existing log files (only useful when running dev mode/local runs).
+func CleanUp(th TestHelper) {
+	t := th.T()
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
	t := th.T()
	t.Helper()
```